lib/content-services/src/lib/common/services/people-content.service.spec.ts

```diff
@@ -16,17 +16,23 @@
  */
 
 import { createNewPersonMock, fakeEcmAdminUser, fakeEcmUser, fakeEcmUser2, fakeEcmUserList } from '../mocks/ecm-user.service.mock';
-import { AlfrescoApiService, AlfrescoApiServiceMock, CoreTestingModule } from '@alfresco/adf-core';
+import { AlfrescoApiService, AlfrescoApiServiceMock, RedirectAuthService } from '@alfresco/adf-core';
 import { PeopleContentQueryRequestModel, PeopleContentService } from './people-content.service';
 import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { EMPTY, of } from 'rxjs';
 
 describe('PeopleContentService', () => {
     let peopleContentService: PeopleContentService;
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [CoreTestingModule],
-            providers: [{ provide: AlfrescoApiService, useClass: AlfrescoApiServiceMock }]
+            imports: [HttpClientTestingModule],
```

Comment:
is there any reason for doing that during migration? the testing module was to save the code, not to have same stuff in each test

Reply:
unit tests were unstable and trying to do actual REST calls , some of them were failing polluting the logs a lot as we discussed that's why some of them are changed